app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,11 @@
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_tag layout: :horizontal, class: 'sign_in',
```

Comment:
Where is this bootstrap_form_tag pointing to? Usually `url:` is right after it, not in the middle of options.